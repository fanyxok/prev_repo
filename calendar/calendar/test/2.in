Shanghaitech 639627 Ih 37
S print_month
G pass_day 48396871
G pass_month -1752607
S go_to 937723 Spst 39
S pass_month 5609625
G print_today
G pass_day -40497354
S print_month
S print_today
G print_today
S print_month
G print_month
S print_year
G print_year
