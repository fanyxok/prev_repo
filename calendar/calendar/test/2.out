Ih                                  
┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │ 01 │ 02 │ 03 │
├────┼────┼────┼────┼────┼────┼────┤
│ 04 │ 05 │ 06 │ 07 │ 08 │ 09 │ 0A │
├────┼────┼────┼────┼────┼────┼────┤
│ 0B │ 0C │ 0D │ 0E │ 0F │ 10 │ 11 │
├────┼────┼────┼────┼────┼────┼────┤
│ 12 │ 13 │ 14 │ 15 │ 16 │ 17 │ 18 │
├────┼────┼────┼────┼────┼────┼────┤
│ 19 │ 1A │ 1B │ 1C │ 1D │ 1E │ 1F │
├────┼────┼────┼────┼────┼────┼────┤
│ 20 │ 21 │ 22 │ 23 │ 24 │ 25 │ 26 │
├────┼────┼────┼────┼────┼────┼────┤
│ 27 │    │    │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘
937447 Jun 1
Sist                                
┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │ 01 │ 02 │
├────┼────┼────┼────┼────┼────┼────┤
│ 03 │ 04 │ 05 │ 06 │ 07 │ 08 │ 09 │
├────┼────┼────┼────┼────┼────┼────┤
│ 0A │ 0B │ 0C │ 0D │ 0E │ 0F │ 10 │
├────┼────┼────┼────┼────┼────┼────┤
│ 11 │ 12 │ 13 │ 14 │ 15 │ 16 │ 17 │
├────┼────┼────┼────┼────┼────┼────┤
│ 18 │ 19 │ 1A │ 1B │ 1C │ 1D │ 1E │
├────┼────┼────┼────┼────┼────┼────┤
│ 1F │ 20 │ 21 │ 22 │ 23 │ 24 │ 25 │
├────┼────┼────┼────┼────┼────┼────┤
│ 26 │ 27 │ 28 │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘
826742 Sist 35
826569 May 7
Sist                                
┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │ 01 │ 02 │
├────┼────┼────┼────┼────┼────┼────┤
│ 03 │ 04 │ 05 │ 06 │ 07 │ 08 │ 09 │
├────┼────┼────┼────┼────┼────┼────┤
│ 0A │ 0B │ 0C │ 0D │ 0E │ 0F │ 10 │
├────┼────┼────┼────┼────┼────┼────┤
│ 11 │ 12 │ 13 │ 14 │ 15 │ 16 │ 17 │
├────┼────┼────┼────┼────┼────┼────┤
│ 18 │ 19 │ 1A │ 1B │ 1C │ 1D │ 1E │
├────┼────┼────┼────┼────┼────┼────┤
│ 1F │ 20 │ 21 │ 22 │ 23 │ 24 │ 25 │
├────┼────┼────┼────┼────┼────┼────┤
│ 26 │ 27 │ 28 │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘
May                                 
┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │ 01 │ 02 │
├────┼────┼────┼────┼────┼────┼────┤
│ 03 │ 04 │ 05 │ 06 │ 07 │ 08 │ 09 │
├────┼────┼────┼────┼────┼────┼────┤
│ 10 │ 11 │ 12 │ 13 │ 14 │ 15 │ 16 │
├────┼────┼────┼────┼────┼────┼────┤
│ 17 │ 18 │ 19 │ 20 │ 21 │ 22 │ 23 │
├────┼────┼────┼────┼────┼────┼────┤
│ 24 │ 25 │ 26 │ 27 │ 28 │ 29 │ 30 │
├────┼────┼────┼────┼────┼────┼────┤
│ 31 │    │    │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘
Sist                                 Spst                                 Slst                                
┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │ 01 │ 02 │ │    │    │    │ 01 │ 02 │ 03 │ 04 │ │ 01 │ 02 │ 03 │ 04 │ 05 │ 06 │ 07 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 03 │ 04 │ 05 │ 06 │ 07 │ 08 │ 09 │ │ 05 │ 06 │ 07 │ 08 │ 09 │ 0A │ 0B │ │ 08 │ 09 │ 0A │ 0B │ 0C │ 0D │ 0E │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 0A │ 0B │ 0C │ 0D │ 0E │ 0F │ 10 │ │ 0C │ 0D │ 0E │ 0F │ 10 │ 11 │ 12 │ │ 0F │ 10 │ 11 │ 12 │ 13 │ 14 │ 15 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 11 │ 12 │ 13 │ 14 │ 15 │ 16 │ 17 │ │ 13 │ 14 │ 15 │ 16 │ 17 │ 18 │ 19 │ │ 16 │ 17 │ 18 │ 19 │ 1A │ 1B │ 1C │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 18 │ 19 │ 1A │ 1B │ 1C │ 1D │ 1E │ │ 1A │ 1B │ 1C │ 1D │ 1E │ 1F │ 20 │ │ 1D │ 1E │ 1F │ 20 │ 21 │ 22 │ 23 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 1F │ 20 │ 21 │ 22 │ 23 │ 24 │ 25 │ │ 21 │ 22 │ 23 │ 24 │ 25 │ 26 │ 27 │ │ 24 │ 25 │ 26 │ 27 │ 28 │ 29 │    │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 26 │ 27 │ 28 │    │    │    │    │ │    │    │    │    │    │    │    │ │    │    │    │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘
Sem                                  Sca                                  Ims                                 
┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │    │ 01 │ │    │    │    │    │ 01 │ 02 │ 03 │ │    │ 01 │ 02 │ 03 │ 04 │ 05 │ 06 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 02 │ 03 │ 04 │ 05 │ 06 │ 07 │ 08 │ │ 04 │ 05 │ 06 │ 07 │ 08 │ 09 │ 0A │ │ 07 │ 08 │ 09 │ 0A │ 0B │ 0C │ 0D │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 09 │ 0A │ 0B │ 0C │ 0D │ 0E │ 0F │ │ 0B │ 0C │ 0D │ 0E │ 0F │ 10 │ 11 │ │ 0E │ 0F │ 10 │ 11 │ 12 │ 13 │ 14 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 10 │ 11 │ 12 │ 13 │ 14 │ 15 │ 16 │ │ 12 │ 13 │ 14 │ 15 │ 16 │ 17 │ 18 │ │ 15 │ 16 │ 17 │ 18 │ 19 │ 1A │ 1B │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 17 │ 18 │ 19 │ 1A │ 1B │ 1C │ 1D │ │ 19 │ 1A │ 1B │ 1C │ 1D │ 1E │ 1F │ │ 1C │ 1D │ 1E │ 1F │ 20 │ 21 │ 22 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 1E │ 1F │ 20 │ 21 │ 22 │ 23 │ 24 │ │ 20 │ 21 │ 22 │ 23 │ 24 │ 25 │ 26 │ │ 23 │ 24 │ 25 │ 26 │ 27 │ 28 │ 29 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 25 │ 26 │ 27 │ 28 │    │    │    │ │ 27 │    │    │    │    │    │    │ │    │    │    │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘
Ihuman                               Siais                                Ih                                  
┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 01 │ 02 │ 03 │ 04 │ 05 │ 06 │ 07 │ │    │    │    │    │    │ 01 │ 02 │ │    │    │ 01 │ 02 │ 03 │ 04 │ 05 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 08 │ 09 │ 0A │ 0B │ 0C │ 0D │ 0E │ │ 03 │ 04 │ 05 │ 06 │ 07 │ 08 │ 09 │ │ 06 │ 07 │ 08 │ 09 │ 0A │ 0B │ 0C │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 0F │ 10 │ 11 │ 12 │ 13 │ 14 │ 15 │ │ 0A │ 0B │ 0C │ 0D │ 0E │ 0F │ 10 │ │ 0D │ 0E │ 0F │ 10 │ 11 │ 12 │ 13 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 16 │ 17 │ 18 │ 19 │ 1A │ 1B │ 1C │ │ 11 │ 12 │ 13 │ 14 │ 15 │ 16 │ 17 │ │ 14 │ 15 │ 16 │ 17 │ 18 │ 19 │ 1A │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 1D │ 1E │ 1F │ 20 │ 21 │ 22 │ 23 │ │ 18 │ 19 │ 1A │ 1B │ 1C │ 1D │ 1E │ │ 1B │ 1C │ 1D │ 1E │ 1F │ 20 │ 21 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 24 │ 25 │ 26 │ 27 │ 28 │    │    │ │ 1F │ 20 │ 21 │ 22 │ 23 │ 24 │ 25 │ │ 22 │ 23 │ 24 │ 25 │ 26 │ 27 │ 28 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │    │    │ │ 26 │ 27 │    │    │    │    │    │ │ 29 │    │    │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘
Jan                                  Feb                                  Mar                                 
┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │ 01 │ 02 │ 03 │ │ 01 │ 02 │ 03 │ 04 │ 05 │ 06 │ 07 │ │ 01 │ 02 │ 03 │ 04 │ 05 │ 06 │ 07 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 04 │ 05 │ 06 │ 07 │ 08 │ 09 │ 10 │ │ 08 │ 09 │ 10 │ 11 │ 12 │ 13 │ 14 │ │ 08 │ 09 │ 10 │ 11 │ 12 │ 13 │ 14 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 11 │ 12 │ 13 │ 14 │ 15 │ 16 │ 17 │ │ 15 │ 16 │ 17 │ 18 │ 19 │ 20 │ 21 │ │ 15 │ 16 │ 17 │ 18 │ 19 │ 20 │ 21 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 18 │ 19 │ 20 │ 21 │ 22 │ 23 │ 24 │ │ 22 │ 23 │ 24 │ 25 │ 26 │ 27 │ 28 │ │ 22 │ 23 │ 24 │ 25 │ 26 │ 27 │ 28 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 25 │ 26 │ 27 │ 28 │ 29 │ 30 │ 31 │ │    │    │    │    │    │    │    │ │ 29 │ 30 │ 31 │    │    │    │    │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │    │    │ │    │    │    │    │    │    │    │ │    │    │    │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘
Apr                                  May                                  Jun                                 
┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │ 01 │ 02 │ 03 │ 04 │ │    │    │    │    │    │ 01 │ 02 │ │    │ 01 │ 02 │ 03 │ 04 │ 05 │ 06 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 05 │ 06 │ 07 │ 08 │ 09 │ 10 │ 11 │ │ 03 │ 04 │ 05 │ 06 │ 07 │ 08 │ 09 │ │ 07 │ 08 │ 09 │ 10 │ 11 │ 12 │ 13 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 12 │ 13 │ 14 │ 15 │ 16 │ 17 │ 18 │ │ 10 │ 11 │ 12 │ 13 │ 14 │ 15 │ 16 │ │ 14 │ 15 │ 16 │ 17 │ 18 │ 19 │ 20 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 19 │ 20 │ 21 │ 22 │ 23 │ 24 │ 25 │ │ 17 │ 18 │ 19 │ 20 │ 21 │ 22 │ 23 │ │ 21 │ 22 │ 23 │ 24 │ 25 │ 26 │ 27 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 26 │ 27 │ 28 │ 29 │ 30 │    │    │ │ 24 │ 25 │ 26 │ 27 │ 28 │ 29 │ 30 │ │ 28 │ 29 │ 30 │    │    │    │    │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │    │    │ │ 31 │    │    │    │    │    │    │ │    │    │    │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘
Jul                                  Aug                                  Sep                                 
┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │ 01 │ 02 │ 03 │ 04 │ │    │    │    │    │    │    │ 01 │ │    │    │ 01 │ 02 │ 03 │ 04 │ 05 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 05 │ 06 │ 07 │ 08 │ 09 │ 10 │ 11 │ │ 02 │ 03 │ 04 │ 05 │ 06 │ 07 │ 08 │ │ 06 │ 07 │ 08 │ 09 │ 10 │ 11 │ 12 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 12 │ 13 │ 14 │ 15 │ 16 │ 17 │ 18 │ │ 09 │ 10 │ 11 │ 12 │ 13 │ 14 │ 15 │ │ 13 │ 14 │ 15 │ 16 │ 17 │ 18 │ 19 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 19 │ 20 │ 21 │ 22 │ 23 │ 24 │ 25 │ │ 16 │ 17 │ 18 │ 19 │ 20 │ 21 │ 22 │ │ 20 │ 21 │ 22 │ 23 │ 24 │ 25 │ 26 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 26 │ 27 │ 28 │ 29 │ 30 │ 31 │    │ │ 23 │ 24 │ 25 │ 26 │ 27 │ 28 │ 29 │ │ 27 │ 28 │ 29 │ 30 │    │    │    │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │    │    │ │ 30 │ 31 │    │    │    │    │    │ │    │    │    │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘
Oct                                  Nov                                  Dec                                 
┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │ 01 │ 02 │ 03 │ │ 01 │ 02 │ 03 │ 04 │ 05 │ 06 │ 07 │ │    │    │ 01 │ 02 │ 03 │ 04 │ 05 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 04 │ 05 │ 06 │ 07 │ 08 │ 09 │ 10 │ │ 08 │ 09 │ 10 │ 11 │ 12 │ 13 │ 14 │ │ 06 │ 07 │ 08 │ 09 │ 10 │ 11 │ 12 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 11 │ 12 │ 13 │ 14 │ 15 │ 16 │ 17 │ │ 15 │ 16 │ 17 │ 18 │ 19 │ 20 │ 21 │ │ 13 │ 14 │ 15 │ 16 │ 17 │ 18 │ 19 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 18 │ 19 │ 20 │ 21 │ 22 │ 23 │ 24 │ │ 22 │ 23 │ 24 │ 25 │ 26 │ 27 │ 28 │ │ 20 │ 21 │ 22 │ 23 │ 24 │ 25 │ 26 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 25 │ 26 │ 27 │ 28 │ 29 │ 30 │ 31 │ │ 29 │ 30 │    │    │    │    │    │ │ 27 │ 28 │ 29 │ 30 │ 31 │    │    │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │    │    │ │    │    │    │    │    │    │    │ │    │    │    │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘
